kodilla-exception/src/test/java/com/kodilla/exception/test/SearchFlight.java

```diff
@@ -0,0 +1,30 @@
+package com.kodilla.exception.test;
+
+import java.util.HashMap;
+
+public class SearchFlight {
+
+    private HashMap<String, Boolean> flightPlan = new HashMap<>();
+
+    public HashMap<String, Boolean> findFlight(Flight flight) {
+
+        flightPlan.put(flight.getArrivalAirport(), true);
+        flightPlan.put(flight.getDepartureAirport(), true);
+        return flightPlan;
+    }
+
+    public void searchFlight(String searchingAirport) throws RouteNotFoundException {
+
+        if (!flightPlan.containsKey(searchingAirport)) {
+            throw new RouteNotFoundException("Airport doesn't exist.");
```

Comment:
which airport does not exist? This information can be crucial

Reply:
as a parameter to constructor, you should have searchingAirport, and exact message should be created inside constructor - thanks to that your message will be coherent - whenever this exception will occur.